test/cppsuite/configs/background_compact_default.txt

```diff
@@ -0,0 +1,61 @@
+# Configuration for background_compact.
+duration_seconds=60,
+cache_size_mb=10000,
+
+timestamp_manager=
```

Comment:
I wonder if this is required?

Reply:
I don't think so, I can remove it.

Reply:
Fixed https://github.com/wiredtiger/wiredtiger/commit/b21630dc0d16a3e94d7d729cc9d519ab60c84c7e.

Reply:
This commit does not contain changes in the configs